crates/language/src/language_settings.rs

```diff
@@ -386,6 +386,8 @@ pub enum ShowWhitespaceSetting {
     None,
     /// Draw all invisible symbols.
     All,
+    /// Draw whitespace only at boundaries.
```

Comment:
> only at boundaries

That's not entirely true though, as tabs are always rendered.

so what if we elaborate more about the tabs rendering and write something like above?
```
// For a whitespace to be on a boundary, any of the following conditions need to be met:
// - It is a tab
// - It is adjacent to an edge (start or end)
// - It is adjacent to a whitespace (left or right)
```

Reply:
Added!